### ISSUE TITLE
blank/empty table

### ISSUE THREAD

**Issue by feodilla** [tabulapdf/tabula#548]
Dear all,
I hope you're doing well. 
It doesn't matter how much I try, my tables are being created but they appear completely empty.
The command shows an issue with the font Times Roman.  
Amsyss-MBP:~ amsys$ cd /Users/amsys/Downloads/TabulaJava 
Amsyss-MBP:TabulaJava amsys$ java -jar tabula-java.jar -g -p all -o tabtest.csv multas-debitos_RJ2016.pdf 
Sep 20, 2016 10:55:12 PM org.apache.pdfbox.pdmodel.font.PDCIDFontType2Font getawtFont
INFO: Can't read the embedded font IIQFIP+Times-Roman
What am I doing wrong?
best
fernanda

**Comment by jazzido** [comment]
Can you send us the PDF, so I can examine the issue for ourselves?

You can send the PDF any of the following three ways:

1. send a link, if the PDF is accessible over the internet
2. attach it via Github's web Issues interface, if the document is not sensitive. (You must use the web interface. If you attempt to attach it to an email reply to GitHub, it will not be sent.)
3. send it directly by email to me (or another Tabula collaborator), using the email address in my profile. If the document is sensitive, please say so.

**Comment by JashDalal** [comment]
Yes, a similar issue is happening with me as well. All my test files in .pdf form when passed to convert_into() and read_pdf() are showing empty dataframes and output csv files.
I have checked the other requirements. It is working for data.pdf(attached in the folder 'example' in this repository)
These pdfs were initially images which are converted by converting images to the pdf format.
Even a screenshot of the "data.pdf" converted to "ssdata.pdf"' is giving a blank dataframe on reading the file.

I have attached the files below:

[aimactek_try.pdf](https://github.com/tabulapdf/tabula/files/3316687/aimactek_try.pdf)
[ssdata.pdf](https://github.com/tabulapdf/tabula/files/3316688/ssdata.pdf)
[tabula_pdf.pdf](https://github.com/tabulapdf/tabula/files/3316689/tabula_pdf.pdf)
[test.pdf](https://github.com/tabulapdf/tabula/files/3316690/test.pdf)

It would be great if a Tabula collaborator could look into it.
Also, could you suggest an alternative way of how we are supposed to use Tabula to extract tables from images since converting them to pdf format is giving an empty table.
